lib/color.ml

```diff
@@ -0,0 +1,26 @@
+type color = int * int * int
+
+(** RGB code for black *)
+let black = (0, 0, 0)
+
+(** RGB code for white *)
+let white = (255, 255, 255)
+
+(** RGB color code for red *)
+let red = (255, 1, 1)
+
+(** RGB color code for green *)
+let green = (1, 255, 1)
+
+(** RGB color code for blue *)
+let blue = (1, 1, 255)
+
+(** RGB color code for yellow *)
+let yellow = (255, 255, 1)
+
+(** RGBA color constant to set transparent background *)
+let transparent = (0, 0, 0, 0)
+
+(** Converts RGB color into opaque RGBA color. 
+    For use w/ `Context.background` *)
+let opaque (r, g, b) = (r, g, b, 255)
```

Comment:
Can we use the same function to create custom colours as well?

Reply:
I'm not sure I understand what you're asking, this function is for setting the background color in the `init` function. To make it easier to use RGB colors that are maybe being used elsewhere as an opaque background color. It wouldn't work for adding `stroke` or `fill` fields because those don't have an alpha field. 

Do you think there should be a color constructor function? I didn't add one because I thought a three-tuple was easy enough to construct.